### PR TITLE
docs: change `prefix` to `suffix` in CONTRIBUTING.md

### LINKED ISSUE #254: [Docs Bug 🐞 report]: `prefix` should be changed to `suffix` in CONTRIBUTING.md
### Describe the bug you found in AsyncAPI Docs.

- As suggested by @derberg [here](https://github.com/asyncapi/website/issues/1599#issuecomment-1520536498), I'm doing as advised.

- The `prefix` word [here](https://github.com/asyncapi/.github/blob/deb094e6ce6c10cf3d2462d14957b1fae30f7cb3/CONTRIBUTING.md?plain=1#L69C47-L69C53) should be changed to `suffix` as per grammatical rules.

- I'll be opening a PR addressing this issue.

### Attach any resources that can help us understand the issue.

Refrence: https://github.com/asyncapi/website/issues/1599

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by AnimeshKumar923**
**Description**

- Changed `prefix` to `suffix` as per grammatical rules.
- Added some more details to be more descriptive.
- Fixes #254

**Related issue(s)**
Fixes #254

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Pull requests should have a title that follows the specification, otherwise, mer
 - `test: ` prefix in the title indicates that PR is only related to tests and there is no need to trigger release.
 - `refactor: ` prefix in the title indicates that PR is only related to refactoring and there is no need to trigger release.
 
-What about MAJOR release? just add `!` to the prefix, like `fix!: ` or `refactor!: `
```
[the remark this answers is not in the document]
Comment (AnimeshKumar923) [reply CONTRIBUTING.md:69]: Yes. We can do it like `add ! to suffix`

[the remark this answers is not in the document]
Comment (derberg) [reply CONTRIBUTING.md:69]: what is suffix for you?

[the remark this answers is not in the document]
Comment (AnimeshKumar923) [reply CONTRIBUTING.md:69]: > what is suffix for you?

As far as I know, `Suffix` means the part that we put/add at the end of a word. (Roughly?)

[the remark this answers is not in the document]
Comment (derberg) [reply CONTRIBUTING.md:69]: > Suffix means the part that we put/add at the end of a word

yes, exactly, and `add ! to suffix` sounds to me like I should add `!` to the suffix, like add it somewhere at the end of the PR title

[the remark this answers is not in the document]
Comment (AnimeshKumar923) [reply CONTRIBUTING.md:69]: > > Suffix means the part that we put/add at the end of a word
> 
> yes, exactly, and `add ! to suffix` sounds to me like I should add `!` to the suffix, like add it somewhere at the end of the PR title

so how about adding this line: `to the suffix after the specification` and we're having two examples at the end of line 69 to clarify what do we want to convey, i.e., `like fix!: or refactor!:`
Thoughts? @derberg

[the remark this answers is not in the document]
Comment (derberg) [reply CONTRIBUTING.md:69]: `to the suffix after the specification`

but again, this is not suffix, 
and also not specification

```
just add `!` to the prefix, like `fix!: ` or `refactor!: 
```

`fix:` is a prefix, this is why instruction says add `!` too prefix. 
you cannot make it more specific, like `just add ! to the end of the prefix` as it is not at the end of the prefix, as `:` is still there

the question is, do we really need to improve anything here if we have clear example in the same sentence `like fix!:  or refactor!:`

[the remark this answers is not in the document]
Comment (AnimeshKumar923) [reply CONTRIBUTING.md:69]: > sounds good, thanks for your patience with me trying to explain things

- Thank you for clearing my confusion.
- Patience is one of my quality. :sparkles: :blush:

**Pull request closed without merging**